libminifi/src/RemoteProcessorGroupPort.cpp

```diff
@@ -360,7 +360,7 @@ std::pair<std::string, int> RemoteProcessorGroupPort::refreshRemoteSite2SiteInfo
           return std::make_pair(host, siteTosite_port_);
         }
       } else {
-        logger_->log_error("Cannot output body to content for ProcessGroup::refreshRemoteSite2SiteInfo: received HTTP code %ll from %s", client->getResponseCode(), fullUrl.str());
+        logger_->log_error("Cannot output body to content for ProcessGroup::refreshRemoteSite2SiteInfo: received HTTP code %lld from %s", client->getResponseCode(), fullUrl.str());
```

Comment:
Use `PRId64` from `<cinttypes>` for `int64_t`. `lld` is meant to be used for `long long int`, which is not guaranteed to be the same as `int64_t`. It is not the same on my platform.

Reply:
fixed by da22085